### PR TITLE
Add meta support for multinomial

### PR DESCRIPTION
**Pull request by drisspg**
# Summary
Found this when trying to compile the text gen loop of nanogpt here: https://github.com/pytorch/benchmark/blob/b33289942ba326703518c819b32c902dabdee8c1/torchbenchmark/models/nanogpt_generate/model.py#L322

cc @soumith @voznesenskym @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### CODE REVIEW

File: torch/_meta_registrations.py [hunk 1]
```diff
@@ -2819,6 +2819,16 @@ def meta_scatter_reduce__two(self, dim, index, src, reduce, include_self=True):
     return self
 
 
+@register_meta([aten.multinomial.default, aten.multinomial.out])
+@out_wrapper()
+def meta_multinomial(input, num_samples, replacement=False, *, generator=None):
+    if input.dim() == 1:
```
Comment (ngimel) [inline torch/_meta_registrations.py:2825]: please add a check that input.ndim < 2

**Review by ngimel — APPROVED** [review]

File: benchmarks/dynamo/common.py [hunk 2]
```diff
@@ -86,7 +86,6 @@ class CI(NamedTuple):
     "tacotron2",
     # torchrec_dlrm requires gcc-11, https://github.com/pytorch/benchmark/pull/1427
     "torchrec_dlrm",
-    "nanogpt_generate",  # invalid multinomial distribution (sum of probabilities <= 0)
```
Comment (drisspg) [inline benchmarks/dynamo/common.py:89]: Is it okay to also make this change. Should I kick off a run before tyring to land this? @voznesenskym @ngimel

Comment (drisspg) [reply benchmarks/dynamo/common.py:89]: Looks like it isnt' erroring but is timing out. I created; https://github.com/pytorch/benchmark/pull/1659. To reduce the number of gen steps

File: .github/ci_commit_pins/torchbench.txt [hunk 3]
```diff
@@ -1 +1 @@
-44ccad1d0647d56fe4e56c0b1933102be5dcc874
```
[the remark this answers is not in the document]
Comment (drisspg) [reply .github/ci_commit_pins/torchbench.txt:1]: Yeah, this new torchbench version lowered the nanogpt gen steps. Before it was timing out. Is there any process to updating the pin needed?

[the remark this answers is not in the document]
Comment (eellison) [reply .github/ci_commit_pins/torchbench.txt:1]: that's fine, just double checking

[the remark this answers is not in the document]
Comment (malfet) [reply .github/ci_commit_pins/torchbench.txt:1]: Do we want to add similar `pinned package` breadcrumbs check to mergebot, similar to the one used for submodule updates?

**Pull request closed without merging**